templates/travel/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,164 @@
+              on="
+                change:AMP.setState({
+                  fields_query: event.value,
+                  fields_query_live: event.value,
+                  fields_query_edited: query_query != event.value
+                });
+                input-debounced:AMP.setState({
+                  fields_query_live: event.value
+                });
+              "
+              value=""
+              [value]="fields_query"
+            >
+            {{> ../icon/map-pin.snip.html}}
+          </label>
+        </div>
+
+        <amp-list
+            class="inline-list clip"
+            layout="flex-item"
+            src="/api/places?types=(regions)&types=(cities)&input="
+            [src]="'/api/places?types=(regions)&types=(cities)&input=' + fields_query_live"
+          >
+            <template type="amp-mustache">
+              {{=[[ ]]=}}
+              <datalist id="locations">
+                {{#predictions}}
+                <option value="{{description}}">
+                {{/predictions}}
+              </datalist>
+              [[={{ }}=]]
+            </template>
+          </amp-list>
+
+        <div class="flex my2 travel-hero-search-dates">
+          <label
+            class="travel-date-input focus"
+            [class]="'travel-date-input focus' + (fields_departure ? ' travel-date-input-touched' : '')"
+          >
+            <input
+              type="date"
+              placeholder="yyyy-mm-dd"
+              pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}"
+              title="yyyy-mm-dd"
+              name="departure"
+              on="
+                change:AMP.setState({
+                  fields_departure: true,
+                  fields_departure_edited: true
+                })
+              "
+            >
+            {{> ../icon/plane-taking-off.snip.html}}
+            <div class="travel-date-input-label">
+              Departure
+            </div>
+          </label>
+          <label
+            class="travel-date-input focus"
+            [class]="'travel-date-input focus' + (fields_return ? ' travel-date-input-touched' : '')"
+          >
+            <input
+              type="date"
+              placeholder="yyyy-mm-dd"
+              pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}"
+              title="yyyy-mm-dd"
+              name="return"
+              on="
+                change:AMP.setState({
+                  fields_return: true,
+                  fields_return_edited: true
+                })
+              "
+              disabled
+              [disabled]="!fields_departure"
+            >
+            {{> ../icon/plane-landing.snip.html}}
+            <div class="travel-date-input-label">
+              Return
+            </div>
+          </label>
+        </div>
+
+      <div class="md-hide lg-hide">
+        <a
+          href="travel-results.amp"
+          class="ampstart-btn btn rounded center travel-semibold white block col-12 bg-purple travel-input-big"
```

Comment:
move travel-* classes first and then bass

DO not use bass colors addon, it would change the UI if the package changes content, also it is not worth it to import it.

Reply:
Good point.